content/workshops/jboss-migration-toolkit/lab2-using-jmt-cli.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Lab 2 - Using RHMAT CLI
+workshops: jboss-migration-toolkit
+workshop_weight: 20
+layout: lab
+---
+
+# About Red Hat Application Migration Toolkit CLI
+The CLI is a command-line tool in the Red Hat Application Migration Toolkit that allows users to assess and prioritize migration and modernization efforts for applications. It provides numerous reports that highlight the analysis results. The CLI is for advanced users with highly customized needs looking for fine-grained control of RHAMT analysis options or to integrate with external automation tools. 
+
+{{% alert warning %}}
+These instructions are only when you are doing this lab outside of the controlled workshop.  The workshop environment has downloaded, installed, and configured your student environments prior to commencement of this workshop session.
+{{% /alert %}}
+
+# Set Up
+1. Install the Java SE Development Kit (JDK) version 8. We recommend using the OpenJDK or the Oracle JDK.
+
+2. Install the RHAMT CLI.
+
+  a. Download the [ZIP archive][1].
```

Comment:
I think you need to add an additional space in front of `a.` and `b.` to get it to render as being under `2.` - currently it's breaking your numbering.